src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -5902,5 +5902,51 @@ static void Main(string[] args)
 
             await TestInRegularAndScriptAsync(code, expected);
         }
+
+        [WorkItem(30207, "http://github.com/dotnet/roslyn/issues/30207")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public async Task TestRecursiveMemberAccess_ForAllOccurrences()
+        {
+            var code =
```

Comment:
I am unable to repro this crash with this repro code, even prior to this fix. I tried on both 15.9 and 16.0 Preview1. What is the easiest way to repro it?

Reply:
This only repro when "introducing local for all occurrences"